ajax/libs/blueimp-gallery/package.json

```diff
@@ -36,27 +36,8 @@
     {
       "basePath": "",
       "files": [
-        "css/blueimp-gallery-indicator.css",
-        "css/blueimp-gallery-video.css",
-        "css/blueimp-gallery.css",
-        "css/blueimp-gallery.min.css",
-        "img/error.png",
-        "img/error.svg",
-        "img/play-pause.png",
-        "img/loading.gif",
-        "img/play-pause.svg",
-        "img/video-play.png",
-        "img/video-play.svg",
-        "js/blueimp-gallery-fullscreen.js",
-        "js/blueimp-gallery-indicator.js",
-        "js/blueimp-gallery-video.js",
-        "js/blueimp-gallery-vimeo.js",
-        "js/blueimp-gallery-youtube.js",
-        "js/blueimp-gallery.js",
-        "js/blueimp-gallery.min.js",
-        "js/blueimp-helper.js",
-        "js/jquery.blueimp-gallery.js",
-        "js/jquery.blueimp-gallery.min.js"
+        "img/*",
+        "+(css|js)/*blueimp*"
```

Comment:
Let's use `+(css|js)/*blueimp*.+(css|map|js)` to avoid .less file ,which appeared before v2.21.3, appears again in the future.